tests/e2e/upgrade/upgrade.wdio-spec.js

```diff
@@ -0,0 +1,70 @@
+const common = require('../../page-objects/common/common.wdio.page');
+const auth = require('../../auth')();
+const utils = require('../../utils');
+
+const { BRANCH, BUILD_NUMBER } = process.env;
+const loginPage = require('../../page-objects/login/login.wdio.page');
+
+const getInstallButton = async () => {
+  for (const element of await $$('span.release-name')) {
+    const text = await element.getText();
+    if (text.startsWith(`${BRANCH} (`)) {
+      const parent = await (await element.parentElement()).parentElement();
+      return await parent.$('.btn-primary');
+    }
+  }
+
+  throw new Error(`Could not find release ${BRANCH}`);
+};
```

Comment:
I tried the text contains selector, but for some reason it threw errors: the query found an element, but when clicking, it threw an error that it's not clickable. It was a very weird error that I couldn't find any fix for on the internet. 

I also thought about adding some specific selector. Because this initial code runs when the base branch is installed, and I really wanted my test to pass _right now_, this was the easier path where I didn't have to update two branches to achieve this.

Reply:
fixed!